src/pages/core/installation.mdx

```diff
@@ -15,17 +15,15 @@ Our only external dependency is Rust, which you need to install for your
 platform.
 
 <Callout>
-  We recommend installing Rust using the official [rustup
-  installer](https://rustup.rs). This makes it easy to stay on the most recent
-  Rust version and to install compiler targets.
+  We recommend installing Rust using the official [rustup installer]. This makes
+  it easy to stay on the most recent Rust version and to install compiler
+  targets.
 </Callout>
 
 <Callout>
-  For production builds you probably also want to install
-  [Docker](https://www.docker.com/), too.
-  <br /> This is because we offer the [CosmWasm Optimizing Compiler](https://github.com/CosmWasm/optimizer),
-  which uses a Docker image to build the smallest contract possible in a deterministic
-  fashion.
+  For production builds you probably also want to install [Docker], too.
```

Comment:
Not sure the best way to mention this, but for development/fast-iteration flows it's often preferable to install `binaryen` (specifically `wasm-opt`) locally and run all the steps without Docker. See e.g. https://github.com/CosmWasm/optimizer/issues/152#issuecomment-2113104183

Anecdotally, I've just found it kinda faster, but it may be a mac thing or I didn't mount the target folder to avoid recompilation, etc. Just adding as a comment, surely doesn't need to block the PR

Reply:
But do you even need this for dev builds? Can't you build with LTO enabled and without stack unwinding for panics and get a bunch of the size savings?

So this cargo config:

```toml
[profile.debug]
lto = "thin"
panic = "abort"
```

But yeah, we should mention that in a follow-up PR. This mainly got highlighted because of PR formatting changes.

Reply:
interesting, I didn't think about changing profiles around too.. yeah, maybe all around "for local iterative dev builds, you might want to [...]" would make for a worthy followup PR :)